create_tag.py

```diff
@@ -255,6 +255,9 @@ def get_parser():
                         action="store_true",
                         default=False,
                         help="Don't actually create the tag, just print the values that would be used.")
+    parser.add_argument("--push-to-remote",
+                        default="origin",
+                        help="Push the tag to this remote (default: 'origin').")
```

Comment:
[major nitpick] I think `--remote` would be just as clear here and much shorter.  It's not that important, it's not the kind of thing we run regularly manually, so nbd.

Reply:
fixed.

Reply:
I also like the simpler `--remote`